### PR TITLE
Add some must have implementations

### PR DESCRIPTION
**Pull request by Taluu**
NEED MOAR PACKAGES FROM PHP-HTTP !!

### CODE REVIEW

**Comment by Taluu** [comment]
this is frozen, because [`message-factory` isn't much implemented...](https://packagist.org/providers/php-http/message-factory-implementation)

**Comment by Taluu** [comment]
cf https://github.com/php-http/message/issues/75

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,10 @@
         "php-http/discovery": "^1.0",
         "php-http/client-common": "^1.4",
         "php-http/message-factory": "^1.0",
-        "php-http/client-implementation": "^1.0"
+        
+        "php-http/client-implementation": "^1.0",
+        "php-http/message-implementation": "^1.0",
```
Comment (Nyholm) [inline composer.json]: This should be psr/http-message-implementation

File: composer.json [hunk 2]
```diff
@@ -29,7 +29,10 @@
...
+        "php-http/message-factory-implementation": "^1.0"
```
Comment (Nyholm) [inline composer.json]: This package does not exist

Comment (Nyholm) [inline composer.json]: Yet...

Comment (Taluu) [inline composer.json]: Hum, looks like it is now ? https://packagist.org/providers/php-http/message-factory-implementation

But that's strange, as there is nothing on the `php-http/message` repo that says that it implements this virtual package, unless I read it wrong ?

File: composer.json (hunk 1 above)
Comment (Taluu) [inline composer.json]: Fixed :}

File: composer.json (hunk 2 above)
Comment (Nyholm) [inline composer.json]: It is not merged yet: https://github.com/php-http/message/pull/80

**Pull request merged**